server/src/main/java/org/bonitasoft/console/common/server/servlet/FileUploadServlet.java

```diff
@@ -134,7 +135,17 @@ public void doPost(final HttpServletRequest request, final HttpServletResponse r
 
             final FileItemFactory fileItemFactory = new DiskFileItemFactory();
             final ServletFileUpload serviceFileUpload = createServletFileUpload(fileItemFactory);
-            final List<FileItem> items = serviceFileUpload.parseRequest(request);
+            if (checkUploadedFileSize) {
+                setUploadSizeMax(serviceFileUpload, request);
+            }
+            List<FileItem> items;
+            try {
+                items = serviceFileUpload.parseRequest(request);
+            } catch (final OutOfMemoryError e) {
+                throw new SizeLimitExceededException("The file exceeds its maximum permitted size.",
```

Comment:
I tried both setter and it the error keeps raising. 
The solution I'm trying to implement is for the servlet to try to alert the user that some error happens. Currently, when this takes place, nothing is thrown back to the browser...